bin/nodejs-dashboard.js

```diff
@@ -22,7 +22,7 @@ program.parse(process.argv);
 
 if (!program.args.length) {
   program.outputHelp();
-  return;
+  process.exit(0); //eslint-disable-line
```

Comment:
This is because of a syntax error babel starts to complain about.
Babel doesn't allow "return outside a function".